kratos/sources/kratos_application.cpp

```diff
@@ -195,6 +196,7 @@ void KratosApplication::RegisterKratosCore() {
     KRATOS_REGISTER_ELEMENT("Element2D3N", mElement2D3N)
     KRATOS_REGISTER_ELEMENT("Element2D6N", mElement2D6N)
     KRATOS_REGISTER_ELEMENT("Element2D4N", mElement2D4N)
+    KRATOS_REGISTER_ELEMENT("Element3D4N_Q", mElement3D4N_Q)
```

Comment:
```suggestion
    KRATOS_REGISTER_ELEMENT("SurfaceElement3D4N", mElement3D4N)
```

then its consistent with the naming of conditions in the core:
https://github.com/KratosMultiphysics/Kratos/blob/df07da955f73c54a554d35e26e6b00165f30f68c/kratos/sources/kratos_application.cpp#L172-L176

Reply:
I don't have a strong opinion on the naming. I guess surfaceElement3D4N can be consistent with conditions, on other hand Quadrilateral3D4N is self-explanatory.

Reply:
I am ok with both versions